Kajabi/sage-lib
# [DSS-73] Responsiveness - Updates responsiveness column

## Description

Updates the Sage docs status page responsiveness column for all components in the responsiveness audit. Any components that are still listed as `todo` will either have a follow-up ticket created or is currently slated for deprecation.

## Testing in `sage-lib`

- Navigate to the [Sage status page](http://localhost:4000/pages/status)
- Check that the responsiveness column has been updated for the majority of components

## Testing in `kajabi-products`

(**LOW**) Updates the Sage docs status page responsiveness column for all components in the responsiveness audit. Sage docs only.


## Related

[DSS-73](https://kajabi.atlassian.net/browse/DSS-73)

[DSS-73]: https://kajabi.atlassian.net/browse/DSS-73?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
